src/api/app/controllers/webui/attribute_controller.rb

```diff
@@ -9,7 +11,10 @@ class Webui::AttributeController < Webui::WebuiController
   helper 'webui/project'
 
   def index
-    @attributes = @container.attribs
+    @attributes = @container.attribs.includes(:issues, :values, attrib_type: [:attrib_namespace]).sort_by(&:fullname)
+
+    # The Bootstrap version is available so far only for packages
```

Comment:
[nitpick] Why is this comment needed? the next line is really explicit and clear :confused:

Reply:
True, I'll remove it